amundsen_application/static/js/components/TableDetail/OpenRequestMetadata/index.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import './styles.scss';
+
+import { GlobalState } from 'ducks/rootReducer';
+import { connect } from 'react-redux';
+import { ToggleRequestAction } from 'ducks/notification/types';
+import { toggleRequest } from 'ducks/notification/reducer';
+import { bindActionCreators } from 'redux';
+
+interface StateFromProps {
+  requestIsOpen: boolean;
+}
+
+export interface DispatchFromProps {
+  toggleRequest: () => ToggleRequestAction;
+}
+
+export type OpenRequestMetadataProps = StateFromProps & DispatchFromProps;
+
+interface OpenRequestMetadataState {}
+
+export class OpenRequestMetadata extends React.Component<OpenRequestMetadataProps, OpenRequestMetadataState> {
+  public static defaultProps: Partial<OpenRequestMetadataProps> = {};
+
+  constructor(props) {
+    super(props);
+  }
+
+  openRequest = () => {
+    if (!this.props.requestIsOpen) {
+      this.props.toggleRequest();
+    }
+  }
+
+  render() {
+    return (
+      <a className={ "open-request" }
```

Comment:
Why would this be an anchor tag? Is it styled in our mock the way that other links are?

Reply:
keeping styling/behavior consistent with add description button